services/api/src/routes/products.js

```diff
@@ -3,6 +3,8 @@ const Joi = require('joi');
 const validate = require('../utils/middleware/validate');
 const { authenticate, fetchUser } = require('../utils/middleware/authenticate');
 const { searchValidation, getSearchQuery, search } = require('../utils/search');
+const Audit = require('../utils/audit');
```

Comment:
I understand you're choosing the word `Audit` here because it's short. But an `AuditTrail` is semantically very different than an `Audit`. An "audit" is really the analysis of an audit trail in order to create a report. So `Audit.create` is semantically bad. We're either:

- Adding to an "audit trail" or
- Appending an "audit trail" or
- Creating an "audit trail entry"

Reply:
What about AuditLog would that be acceptable?

Reply:
+1 for `AuditEntry` for me... yes it's semi-non-standard pluralization but that shouldn't be a barrier